deps/npm/node_modules/node-gyp/lib/install.js

```diff
@@ -470,11 +470,11 @@ function getDefaultIojsUrl(version) {
 if (require.main === module) {
   var assert = require('assert')
   console.log('test v2.3.4 -> https://iojs.org/download/release')
-  assert(getDefaultIojsUrl('v2.3.4', 'https://iojs.org/download/release'))
+  assert.equal(getDefaultIojsUrl('v2.3.4'), 'https://iojs.org/download/release')
   console.log('test v2.3.4-nightly12345678aaaaaaaaaa -> https://iojs.org/download/nightly')
-  assert(getDefaultIojsUrl('v2.3.4-nightly12345678aaaaaaaaaa', 'https://iojs.org/download/nightly'))
+  assert.equal(getDefaultIojsUrl('v2.3.4-nightly12345678aaaaaaaaaa'), 'https://iojs.org/download/nightly')
```

Comment:
Just an observation that the dot that was appended to the rc format might look good on nightly and next-nightly too. Eg: v2.3.4-nightly.12345678aaaaaaaaaa